### DISCUSSION TITLE
Basset error on "php artisan backpack:install"

### DISCUSSION THREAD

**Discussion by jimcraner** [Laravel-Backpack/community-forum#811] [Q&A (Help)] [answered]
I have a fresh Laravel 10.41.0 install with Breeze and nothing else.  I downloaded backpack/crud 6.5 via composer but when I try to run the backpack:install command, I consistently get this error:

`Initializing basset check ..................................................................................... DONE  
  Checking cache storage ........................................................................................ DONE  
  Fetching a basset ............................................................................................ ERROR  

   ERROR  Error fetching the file.  

  │ Backpack Basset failed to check it's working properly.`

I've tried the following in various combinations:

* confirmed APP_URL is set correctly
* server IS running and accessible at http://localhost
* public disk is configured and accessible
* storage symlink created
* reinstalling backpack
* manually installing basset via composer
* basset:install, basset:check, and basset:clear ALL return the same error as backpack:install
* optimize:clear
* switching APP_URL to https://localhost instead of http:// because I saw that mentioned in another issue (basset still fails with a curl "failed to connect to localhost port 443" error of course)
* changing the .env "BASSET_DEV_MODE" value to true and false and removing it completely
* searched the issue queue for "basset"

I would be grateful for any other suggestions.  Thanks!

**Comment by karandatwani92** [comment, marked as answer]
Hi @jimcraner 

I tried the same to confirm and I faced the same error.
At first, I thought it was a BUG. It's NOT.

There was a step we both missed. I think we should highlight it in the docs so others don't face it.
Let's do a recap:
1. I did a fresh laravel installation
2. Updated DB config in .env
3. Executed `composer require backpack/crud`
4. Executed `php artisan serve` and got `http://127.0.0.1:8000`  access URL.
5. Updated APP_URL with the access URL. We need to put the URL from where the application is currently accessible  **<-- We both missed this step**
6. Finally ran `php artisan backpack:install` which started working.

Thanks for reporting, let me know if you still face the issue.

**Comment by jimcraner** [comment]
Hi @karandatwani92 -- you are correct, that solved it.  I can't believe I missed it.  But at least I wasn't alone  :-)   

Thanks a lot! Have a great day!
